File: Python/SQLAlchemy_ORM/dal/entities/territory.py
from sqlalchemy import Column, ForeignKey, Integer, String
from sqlalchemy.orm import relationship
from dal.data.base import Base

class Territory(Base):
    __tablename__ = 'Territories'

    TerritoryID = Column(String(20), primary_key=True)
    TerritoryDescription = Column(String(50))
    RegionID = Column(Integer, ForeignKey('Region.RegionID'))

    region = relationship('Region', back_populates='territories')
    employees = relationship('Employee', secondary='EmployeeTerritories', back_populates='territories')

File: Go/GORM_ORM/dal/entities/territory.go
package entities

type Territory struct {
	TerritoryID          string `gorm:"primaryKey;size:20;column:TerritoryID"`
	TerritoryDescription string `gorm:"size:50;column:TerritoryDescription"`
	RegionID             int    `gorm:"column:RegionID"`
	Region               Region `gorm:"foreignKey:RegionID"`
	EmployeeTerritories  []EmployeeTerritory
}
